### PR TITLE
ng ctrl exercise

### PR DESCRIPTION
**Pull request by JoeShep**
A new exercise that requires boostrapping an Angular app with a simple single controller. I plan to make this a short series of exercises that will build on each other, increasing the complexity of the app; the next one will move the data interaction to a factory; the final one will require moving the markup into a partial and implementing routing.

Please offer opinions on whether having them setup the full task runner portion of the app is overkill or proper expectation. Also wondering if we have them use bower or npm to install Angular, or offer them the choice

### CODE REVIEW

**Comment by chortlehoort** [comment]
Why are there two files?

**Comment by JoeShep** [comment]
D'oh. I renamed the file and didn't pull the original one

File: 5-rich-browser-applications/exercises/WIDE_WORLD_TRAVEL_1.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#### Working With Angular
+### Wide World Travel Guides -- part 1: Using a controller
+
+For this project you need to use Angular to list a collection of travel guides in the DOM.
+
+----
```
Comment (chortlehoort) [inline 5-rich-browser-applications/exercises/WIDE_WORLD_TRAVEL_1.md]: Add a clear Requirements section here where you explicitly state what the students should be doing.

For example:

Your job is to create a basic Angular application that has one view, and one controller that will be responsible for retrieving the data and exposing it for use in the view. The view needs to list all book titles, which type of book it is, and the price for the book.

**Comment by JoeShep** [comment]
@chortlehoort

**Comment by chortlehoort** [comment]
@JoeShep 👍

**Pull request merged**